code/components/jomjol_flowcontroll/ClassFlowControll.cpp

```diff
@@ -927,3 +927,26 @@ string ClassFlowControll::getJSON()
 {
     return flowpostprocessing->GetJSON();
 }
+
+/** 
+ * Generate the MetricFamily from all available sequences
+ * @returns the string containing the text wire format of the MetricFamily
+ **/
+string ClassFlowControll::getOpenMetrics(string prefix)
+{
+    string res = "";
+    std::vector<NumberPost*> *numbers = flowpostprocessing->GetNumbers();
+    
+    for (int i = 0; i < (*numbers).size(); ++i) {
+        // only valid data is reported (https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#missing-data)
+        if((*numbers)[i]->ReturnValue.length() > 0) {
+            res += prefix + "flow_value{sequence=\"" + (*numbers)[i]->name + "\"} " + (*numbers)[i]->ReturnValue + "\n";
```

Comment:
The name name may contain literal `"`, requiring escaping (something converting to JSON should be good enough).